documentation/source/users/arkane/input.rst

```diff
@@ -204,7 +204,8 @@ Allowed bond types for the ``bonds`` parameter are, e.g., ``'C-H'``, ``'C-C'``,
 
 ``'O=S=O'`` is also allowed.
 
-The order of elements in the bond correction label is not important. Use ``-``/``=``/``#`` to denote a
+The order of elements in the bond correction label is important. The first atom
```

Comment:
[This line](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/arkane/statmech.py#L943) also checks the reverse of the string for the bond correction. I thought it worked fine. Did you experience problems reading bond corrections in reverse? If so, let's fix it (but keep the documentation)

Reply:
If this is the case then, I should change [this line](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/arkane/statmech.py#L943) to avoid confusion. 

It is minor, but one bond correction for 'cbs-qb3-paraskevas' leads to unexpected behavior where reversing the order doesn't match:

```
>>> symbol = 'Cl-C'
>>> symbol[::-1]
'C-lC'
```

Should I change the commented line with the incorrect information?

Reply:
Yes, this is indeed a bug. Could you change it so it works for atoms with more than one char long symbols? Good catch

Reply:
It seems like @cgrambow's PR already fixed this issue (using algorithm from the updated master branch):

```
>>> import re
>>> symbol = 'C=Cl'
>>> ''.join(re.findall('[a-zA-Z]+|[^a-zA-Z]+', symbol)[::-1])
'Cl=C'
```